### DISCUSSION TITLE
kairship/failover/optimization-failover/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2027] [Announcements]
# kairship/failover/optimization-failover/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kairship/failover/optimization-failover/

**Comment by mark8s** [comment]
请问实现原理是什么？底层是istio？ 能稍微详细的描述以下么
